horace_core/sqw/@SQWDnDBase/SQWDnDBase.m

```diff
@@ -22,6 +23,9 @@
         %     %   Detailed explanation goes here
         %     outputArg = obj.Property1 + inputArg;
         % end
+        wout = IX_dataset_1d (w);
```

Comment:
It's probably a good opportunity to remove the placeholder function comments from the public and protected methods blocks.

Reply:
It will be a good opportunity when the PR passes its tests.